bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/IntegrationTests.java

```diff
@@ -56,7 +56,7 @@
     TestScan.class,
     TestSnapshots.class,
     TestIncrement.class,
-    TestListTables.class,
+    TestListTablesHBase2.class,
```

Comment:
Please also include TestListTables

Reply:
TestListTables is already part of Hbase1.x integration test.